app/routes/__index/index.tsx

```diff
@@ -13,7 +13,7 @@ export const loader = async ({ request }: LoaderArgs) => {
 
 export default function IndexRoute() {
   return (
-    <VStack alignSelf="center" height="full">
+    <VStack alignSelf="center" height="94vh">
```

Comment:
Why this magic number?

Reply:
I forgot to remove it, I had it set before to have a margin for the seneca policy text otherwise it will be too close to the cards, but I decided to go with padding instead